packages/main/src/plugin/kubernetes/context-health-checker.ts

```diff
@@ -73,6 +73,10 @@ export class ContextHealthChecker implements Disposable {
     });
   }
 
+  get contextName(): string {
```

Comment:
This is required for testing purpose, kinda very annoying to be force to add a method only used by tests, but currently in vitest v3 it does some magic dark tricks, that is not supported anymore, and the only way I found to make it works is to expose the contextName;

This is not really a big deal, as a lot of classes already expose this info in readyonly mode